Magento_Checkout/web/template/minicart/item/default.html

```diff
@@ -34,7 +34,6 @@ <h3 class="minicart-product__name">
                             "
                             aria-expanded="false"
                             aria-controls="see-details"
-                            data-dropdown="see-details"
                         >
                             <!-- ko i18n: 'See Details' --><!-- /ko -->
                             <svg class="icon dropdown-list__icon">
```

Comment:
plaese add title for svg and role attr like here https://github.com/SnowdogApps/magento2-alpaca-components/tree/master/components/01-globals/icons

Reply:
added